README.md

```diff
@@ -56,7 +57,7 @@ If you need, you can also use the underlying implementation which does not keep
 track of the previously slugged strings (not recommended):
 
 ```js
-var slug = require('github-slugger').slug;
+import GithubSlugger, {slug} from 'github-slugger'
```

Comment:
`GithubSlugger` as a variable is unused in this example, but folks sometimes don’t know you can get the default *and* a named export, so I think it’s good to show that here